_overviews/jdk-compatibility/overview.md

```diff
@@ -4,70 +4,79 @@ title: JDK Compatibility
 permalink: /overviews/jdk-compatibility/overview.html
 ---
 
-Scala runs primarily on the Java Virtual Machine (JVM).
+Scala's primary platform is the Java Virtual Machine (JVM). (Other supported platforms: [Scala.js](https://www.scala-js.org/), [Scala Native](https://scala-native.readthedocs.io/).)
 
-Sometimes new JVM versions require us to update Scala to remain compatible.
-
-And as Scala and the JVM improve independently over time, Scala may drop compatibility with older JVM versions, in order to better take advantage of new JVM features.
+Sometimes new JVM and JDK (Java Development Kit) versions require us to update Scala to remain compatible.
 
 ## Version compatibility table
 
-This table shows the first Scala release in each series that works with each JVM release.
-
-| JVM version | Minimum Scala versions                                                                        |
+| JDK version | Minimum Scala versions           | Recommended Scala versions                                 |
 |:-----------:|:----------------------------------------------------------------------------------------------|
-| 13, 14      | 2.13.1, 2.12.9, see [below](#jdk-13-and-14-compatibility-notes) for further notes                    |
-| 12          | see [below](#jdk-12-compatibility-notes)                                                      |
-| 11          | 2.13.0, 2.12.4, 2.11.12, 2.10.7 (but also, see [below](#jdk-11-compatibility-notes))          |
-| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2                                                                |
-| 7           | 2.11.0, 2.10.0                                                                                |
-| 6           | 2.11.0, 2.10.0                                                                                |
+| 13, 14      | 2.13.2, 2.12.11                  | 2.13.2, 2.12.11                                            |
+| 12          | 2.13.1, 2.12.9                   | 2.13.2, 2.12.11                                            |
+| 11          | 2.13.0, 2.12.4, 2.11.12          | 2.13.2, 2.12.11, 2.11.12                                   |
+| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2   | 2.13.2, 2.12.11, 2.11.12, 2.10.7                           |
+| 6, 7        | 2.11.0, 2.10.0                   | 2.11.12, 2.10.7                                            |
+
+Even when a version combination isn't listed as supported, most features may still work.  (But Scala 2.12+ definitely doesn't work at all on JDK 6 or 7.)
+
+In general, Scala works on JDK 11+, including GraalVM, but it might not take advantage of features that were added after JDK 8. See[below](#jdk-11-compatibility-notes).
```

Comment:
```suggestion
In general, Scala works on JDK 11+, including GraalVM, but it might not take advantage of features that were added after JDK 8. See [below](#jdk-11-compatibility-notes).
```